libraries/WebServer/src/WebServer.cpp

```diff
@@ -23,14 +23,17 @@
 
 #include <Arduino.h>
 #include <esp32-hal-log.h>
+#include <libb64/cdecode.h>
 #include <libb64/cencode.h>
 #include "WiFiServer.h"
 #include "WiFiClient.h"
 #include "WebServer.h"
+#include "HEXBuilder.h"
 #include "FS.h"
 #include "detail/RequestHandlersImpl.h"
 #include "mbedtls/md5.h"
-
+#include "mbedtls/sha1.h"
+#include "mbedtls/base64.h"
```

Comment:
As @Jason2866 said, I don't think it is a good idea to the WebServer to depend on mbedtls. Maybe a minimal version of the functions needed could be provided ?